app/src/pages/inside/profilePage/accessTokenBlock/accessTokenBlock.jsx

```diff
@@ -44,6 +45,14 @@ const messages = defineMessages({
     defaultMessage:
       'In order to provide security for your own domain password, you can use a user token - to verify your account to be able to log with agent.',
   },
+  warningHeader: {
+    id: 'AccessTokenBlock.warningHeader',
+    defaultMessage: 'Keep the token safe!',
+  },
+  warning: {
+    id: 'AccessTokenBlock.warning',
+    defaultMessage: "This token shouldn't be shared or published at any type of public sources.",
```

Comment:
Could you please add these messages for other localizations

Reply:
Added